internal/integration/json_helpers_test.go

```diff
@@ -378,8 +372,7 @@ func createTransactionOptions(t testing.TB, opts bson.Raw) *options.TransactionO
 		case "readConcern":
 			txnOpts.SetReadConcern(createReadConcern(opt))
 		case "maxCommitTimeMS":
-			t := time.Duration(opt.Int32()) * time.Millisecond
-			txnOpts.SetMaxCommitTime(&t)
+			t.Skip("GODRIVER-2348: maxCommitTimeMS is deprecated")
```

Comment:
This option was removed per the CSOT specifications concerning major version updates. We can't fully remove this use case until DRIVERS-2829 is completed.